OpenUpSA/wazimap-ng-ui
# design changes of add filter button

## Description
modifying choropleth add filter button

## Related Issue
https://trello.com/c/f3ogmTOu/841-common-add-filter-button-for-data-mapper-and-rich-data-view-people-are-finding-it-easy-to-use-the-data-mapper-add-filter-button


## How to test it locally
* select any subindicator from the data mapper
* when the choropleth is displayed, confirm that instead of `add filter button`, there is a `+` button  and it is working

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/125350275-38433f80-e367-11eb-80e1-7466dd6f73b0.png)


## Changelog

### Added

### Updated
* `index.html`, `webflow.js`, `webflow.css` to apply the design changes
* `mapchip.js` to update the code that manages the button

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
